tests/common/helpers/platform_api/chassis.py

```diff
@@ -169,3 +171,14 @@ def get_watchdog(conn):
 
 def get_eeprom(conn):
     return chassis_api(conn, 'get_eeprom')
+
+
+def get_supervisor_slot(conn):
+    return chassis_api(conn, 'get_supervisor_slot')
+
+
+def get_my_slot(conn):
+    return chassis_api(conn, 'get_my_slot')
+
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008/), there should be two blank lines between top-level function definitions.